### PR TITLE
Added dockercompose + changed to unless-stopped

### PR DESCRIPTION
**Pull request by exetico**
I prefer unless-stopped, but all guides are spammed with "always" - i guess that's why it's used :)?

Also, docker compose if someone can use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,11 +50,34 @@ This project is a _Web GUI_ for the very good [Zigbee2Mqtt](https://www.zigbee2m
...
+#########################
+# zigbee2mqttAssistant (Interface)
+#########################
+# https://github.com/yllibed/Zigbee2MqttAssistant
+
+  zigbee2mqttAssistant:
+    image: carldebilly/zigbee2mqttassistant
+    container_name: zigbee2mqttAssistant
+    environment:
+      - Z2MA_SETTINGS__MQTTSERVER={IP}
```
Comment (carldebilly) [inline README.md]:
```suggestion
      - Z2MA_SETTINGS__MQTTSERVER={IP_OR_HOSTNAME}
```

**Comment by carldebilly** [comment]
@all-contributors please add exetico to code and doc

File: README.md (hunk 1 above)
Comment (exetico) [reply README.md]: Perfect (Y)

**Comment by exetico** [comment]
Thanks for accepting the PR. I'll come with more input if needed, in the future :-)

**Pull request merged**